asset-transfer-basic/rest-api-typescript/package-lock.json

```diff
@@ -0,0 +1,6830 @@
+{
```

Comment:
The is an application rather than a library so I think checking in package-lock.json is the right thing to do

Reply:
For every other application sample, we have decided not to check in package-lock.json since it gets auto-generated for people with the latest dependencies.

Reply:
That doesn't seem like a good example to set, and the rest sample uses `npm ci` in the docker image